src/applications/vaos/tests/appointment-list/components/AppointmentsPage/AppointmentsPageV2.unit.spec.jsx

```diff
@@ -407,7 +407,9 @@ describe('VAOS <AppointmentsPageV2>', () => {
       }),
     );
 
-    expect(await screen.findAllByRole('radio')).to.have.length(2);
+    await waitFor(() => {
+      expect(screen.getAllByRole('radio')).to.have.length(2);
```

Comment:
Fixes a flaky test, because we were only waiting for `findAllByRole` to return something, not that it returned 2 items.